dotCMS/src/integration-test/java/com/dotcms/security/apps/AppsAPIImplTest.java

```diff
@@ -434,6 +439,104 @@ public void Test_Secret_Destroy_Method() {
         }
     }
 
+    /**
+     * Test computeSecretWarnings.
+     * Given scenario: You create a secret with no value the secret is expected to be required according to the descriptor.
+     * Expected Result: Such situation should generate a warning
+     */
+    @Test
+    public void Test_Secret_Integrity_Check_Expect_Warning()
+            throws DotDataException, DotSecurityException {
+        final AppsAPI api = APILocator.getAppsAPI();
+
+        final String appKey = "any-key";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "any-key" appears 15 times in this file; the first occurrence is on line 451](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5519512)